### PR TITLE
Selenium4

### PR DESCRIPTION
**Pull request by jtnord**
testing and adaptation to https://github.com/jenkinsci/acceptance-test-harness/pull/1100

I am unable to test this as despite the fact that the ATH works perfectly for me on windows and with a containerized remote webdriver and docker support - the "selenium in a container" mode does not work.

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: ui-tests/pom.xml [hunk 1]
```diff
@@ -33,14 +33,29 @@
         <artifactId>jenkins-core</artifactId>
         <version>${jenkins.version}</version>
       </dependency>
+	<!-- need to match any BOM imports from ATH due to https://issues.apache.org/jira/browse/MNG-5761 -->
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-bom</artifactId>
+        <version>4.1.91.Final</version>
```
Comment (jtnord) [inline ui-tests/pom.xml:40]: version was lower than required in #1470  care needs to be taken when updating acceptnace-test-harness to keep these aligned (or higher) due to  https://issues.apache.org/jira/browse/MNG-5761

File: ui-tests/pom.xml [hunk 2]
```diff
@@ -33,14 +33,29 @@
...
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
```
Comment (jtnord) [inline ui-tests/pom.xml:44]: there was a mix of different jackson versions (core databind and some others) that caused CNFE.

File: ui-tests/pom.xml [hunk 3]
```diff
@@ -33,14 +33,29 @@
...
+      </dependency>
     </dependencies>
   </dependencyManagement>
 
   <dependencies>
     <dependency>
       <groupId>org.jenkins-ci</groupId>
       <artifactId>acceptance-test-harness</artifactId>
-      <version>5504.v485694f31cdf</version>
+      <version>5535.v48093a_7cc4a_c</version>
```
Comment (jtnord) [inline ui-tests/pom.xml:58]: updated version https://github.com/jenkinsci/acceptance-test-harness/pull/1101
```suggestion
      <version>5536.v62b_b_4b_a_49e65/</version>
```

**Comment by uhafner** [comment]
Integrated in #1470

**Pull request closed without merging**